portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,25 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+/**
+ * Adds a random cat picture to the page.
+ */
+function addRandomCat() {
+  const cats =
+      ["/images/kitten-in-bed.jpg", "/images/sleepy-kitten.jpg"];
+
+  // Pick a random greeting.
+  const cat = cats[Math.floor(Math.random() * cats.length)];
+
+  // Create img element
```

Comment:
nit: For comments now, in the future, make full sentences with the period at the end.